source/_integrations/zha.markdown

```diff
@@ -115,6 +115,8 @@ it is expected to see `NCP entered failed state. Requesting APP controller resta
 
 </div>
 
+{% include integrations/config_flow.md %}
+
 ## Configuration - GUI
```

Comment:
```suggestion
{% include integrations/config_flow.md %}

## Configuration - GUI
```

This will create duplicate configuration headings. The instructions below seem to be correct and more fitting that the generic config flow templates.

Adding this for a `my` button, seems therefore incorrect.


Closing this contribution for the above reasoning.

../Frenck